src/index.ts

```diff
@@ -7,7 +7,7 @@ import { randomBytes } from "crypto";
 
 export function hermes(options?: { hermesPath?: string }): Plugin {
     options ??= {};
-    options.hermesPath ??= join(__dirname, "..", "..", "hermes-engine"); // node_modules/rollup-plugin-hermes/dist/../../hermes-engine
+    options.hermesPath ??= join(__dirname, "../../@aliucord/hermes"); // node_modules/@aliucord/hermes
```

Comment:
Just keep the context
```suggestion
    options.hermesPath ??= join(__dirname, "../../@aliucord/hermes"); // node_modules/rollup-plugin-hermes/dist/../../@aliucord/hermes
```